src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.tsx

```diff
@@ -346,25 +346,41 @@ export class EditDataSourceForm extends React.Component<EditDataSourceProps, Edi
 
   onClickTestConnection = async () => {
     this.setState({ isLoading: true });
-    const existingAuthType = this.props.existingDataSource.auth.type;
+    // const existingAuthType = this.props.existingDataSource.auth.type;
```

Comment:
nit: let's remove this unused piece of commented code

Reply:
good catch